config/manifests/community/bases/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     capabilities: Deep Insights
     categories: Logging & Tracing,Monitoring
     certified: "false"
-    containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
```

Comment:
Why is this being changed?

Reply:
I can revert if you want because I don't have a strong opinion on this but I saw that other operators are versioning the image and it looks to me as a good practice that makes sense since the image is associated to a version.

Reply:
I'd rather it be done in a follow up, but I'm ok with it.